packages/shared/src/graphql/feed.ts

```diff
@@ -22,6 +22,11 @@ export const FEED_POST_FRAGMENT = gql`
       name
       image
     }
+    sharedPost {
```

Comment:
This does not cause any issues on other source feeds and just returns null. But we can also make it conditional if you want.